File: app/src/main/java/com/iug/coursehub/feature/user/home_screen/UserHomeNavHostFragment.java
package com.iug.coursehub.feature.user.home_screen;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.iug.coursehub.databinding.FragmentUserHomeNavHostBinding;

public class UserHomeNavHostFragment extends Fragment {

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        FragmentUserHomeNavHostBinding binding = FragmentUserHomeNavHostBinding.inflate(inflater, container, false);



        return binding.getRoot();
    }
}

File: app/src/main/java/com/iug/coursehub/feature/user/home_screen/UserHomeNavHostFragment.kt
package com.iug.coursehub.feature.user.home_screen

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.iug.coursehub.R
import com.iug.coursehub.databinding.FragmentUserHomeNavHostBinding
import com.iug.coursehub.feature.user.home_screen.cuorse_categories_screen.CategoriesPagerAdapter

class UserHomeNavHostFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        val binding = FragmentUserHomeNavHostBinding.inflate(inflater, container, false)

        binding.apply {

        }

        return binding.root
    }
}